assets/src/amp-story-editor-blocks.js

```diff
@@ -163,7 +166,7 @@ const setBlockParent = ( props ) => {
 		};
```

Comment:
^ `ALLOWED_BLOCKS` within `setBlockParent` would also need switching for `ALLOWED_CHILD_BLOCKS` -- otherwise AMP Story Page requires another Page to be its parent and it won't be possible to insert it to the root level (and probably would be possible within another Page, setting `parent` overrides the `allowedBlocks` of `InnerBlocks`).

Reply:
^ That should fix it!

> Ugh, looks like one can still insert pages inside pages, even though it's forbidden with `allowedBlocks`.

Reply:
Yay! So it was just an oversight from my side, phew :-)